web-registry/src/pages/Home/Home.loader.ts

```diff
@@ -0,0 +1,74 @@
+import { QueryClient } from '@tanstack/react-query';
+
+import { getEcocreditQueryClient } from 'lib/clients/ecocreditQueryClient';
+import { getMarketplaceQueryClient } from 'lib/clients/marketplaceQueryClient';
+import { getSimplePriceQuery } from 'lib/queries/react-query/coingecko/simplePrice/simplePriceQuery';
+import { getProjectsQuery } from 'lib/queries/react-query/ecocredit/getProjectsQuery/getProjectsQuery';
+import { getAllowedDenomQuery } from 'lib/queries/react-query/marketplace/getAllowedDenomQuery/getAllowedDenomQuery';
+import { getSellOrdersExtentedQuery } from 'lib/queries/react-query/marketplace/getSellOrdersExtentedQuery/getSellOrdersExtentedQuery';
+import { getAllCreditClassesQuery } from 'lib/queries/react-query/sanity/getAllCreditClassesQuery/getAllCreditClassesQuery';
+import { getAllHomePageQuery } from 'lib/queries/react-query/sanity/getAllHomePageQuery/getAllHomePageQuery';
+import { getFromCacheOrFetch } from 'lib/queries/react-query/utils/getFromCacheOrFetch';
+
+import { client as sanityClient } from '../../sanity';
+
+type LoaderType = {
+  queryClient: QueryClient;
+};
+
+export const homeLoader =
+  ({ queryClient }: LoaderType) =>
+  async ({ params }: { params: any }) => {
+    // Clients
+    const ecocreditClient = await getEcocreditQueryClient();
+    const marketplaceClient = await getMarketplaceQueryClient();
+
+    // Queries
+    const allHomePageQuery = getAllHomePageQuery({ sanityClient });
+    const allCreditClassesQuery = getAllCreditClassesQuery({ sanityClient });
+    const simplePriceQuery = getSimplePriceQuery({});
+    const projectsQuery = getProjectsQuery({
+      client: ecocreditClient,
+      request: {},
```

Comment:
do you think it would be an improvement to make an empty `{}` the default value for `request` and make it optional to pass? at least here the call would be marginally simpler

Reply:
It would but I'm reusing existing ones for most queries, changing that would involve a lot of changes at once.